docs/architecture.md

```diff
@@ -0,0 +1,86 @@
+### Architecture
+
+Feather uses the Entity-Component-System architecture, also known as ECS. This architecture
+is widely used in the Rust gamedev ecosystem. For more information, we recommend checking
+out:
+* the [`hecs`](https://docs.rs/hecs) documentation
+* the [Bevy documentation](https://bevyengine.org/learn/book/getting-started/ecs/)
+
+The Feather game state is defined in the `Game` struct, which lives in `crates/common/src/game.rs`.
+This struct contains the `World` (blocks) and the `Ecs` (entities). It also provides
```

Comment:
`Ecs` (Entitnes), remember Minecraft also have entities, maybe clarify that this is a superset of a Minecraft entity.